packages/docs-reanimated/docs/layout-animations/layout-transitions.mdx

```diff
@@ -4,195 +4,328 @@ sidebar_position: 3
+<InteractiveExample src={LayoutTransitionsSrc} component={LayoutTransitions} />
 
-## How to use predefined layout transition?
+## Predefined Transitions
+
+Below we listed all of the currently available predefined layout transitions with their [layout animation modifiers](/docs/fundamentals/glossary#layout-animation-modifier).
+
+## Linear transition
+
+Linear transition animates both position and dimension in the same way.
 
-### 1. Import chosen transition
+### Reference
 
 ```js
-// LinearTransition is just an example and should be replaced by real animation. For Instance Layout
 import { LinearTransition } from 'react-native-reanimated';
+
+function App() {
+  return <Animated.View layout={LinearTransition} />;
+}
 ```
 
-### 2. Choose Animated Component which layout you want to animate
+### Modifiers
 
-```js
-    // AnimatedComponent - component created by createAnimatedComponent or imported from Reanimated
-    <AnimatedComponent layout={LinearTransition} >
+#### Easing <Optional/>
+
+Time-based modifiers relay on [`withTiming`](/docs/animations/withTiming) function.
+
+```javascript
+LinearTransition.easing(Easing.ease);
+```
+
+- `.easing(easingFunction: EasingFunction)` is an easing function which defines the animation curve. Defaults to `Easing.inOut(Easing.quad)`
+
+:::note
+The `.easing(...)` modifier have no effect when `.springify()` is used.
+:::
+
+#### Spring-based <Optional/>
+
+Spring-based modifiers relay on [`withSpring`](/docs/animations/withSpring) function.
+Just as in withSpring config, spring-based modifiers can be physics-based (has `mass` and `damping` modifiers) or duration-based (has `duration` and `dampingRatio` modifiers).
+
+:::info
+The `mass` and `damping` (physics-based) properties can't be used at the same time as `duration` and `dampingRatio` (duration-based).
+
+When used together `duration` and `dampingRatio` overrides `mass` and `damping` props.
+:::
+
+```javascript
+LinearTransition.springify()
+  .damping(30)
+  .mass(5)
+  .rotate(20)
+  .stiffness(10)
+  .overshootClamping(false)
+  .restDisplacementThreshold(0.1)
+  .restSpeedThreshold(5);
+```
+
+- `.springify()` enables the spring-based animation configuration.
+- `.damping(value: number)` decides how quickly a spring stops moving. Higher damping means the spring will come to rest faster. Defaults to `10`.
+- `.dampingRatio(value: number)` decides how damped the spring is. Value `1` means the spring is critically dmaped, and value `>1` means the spring is overdamped. Defaults to `0.5`.
+- `.mass(value: number)` is the weight of the spring. Reducing this value makes the animation faster. Defaults to `1`.
+- `.rotate(degree: string)` lets you rotate the element.
+- `.stiffness(value: number)` decides how bouncy the spring is. Defaults to `100`.
```

Comment:
More info needed. Does it mean that increasing stiffness increases bounciness of the spring, or the other way? I'd add that clarification